ld-eventsource.gemspec

```diff
@@ -27,4 +27,5 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "concurrent-ruby", "~> 1.0"
   spec.add_runtime_dependency "http_tools", '~> 0.4.5'
```

Comment:
I don't think we still need `http_tools`, do we? It was being used to parse HTTP headers.